common/src/main/java/net/dodogang/plume/ash/client/registry/BlockEntityRendererRegistry.java

```diff
@@ -18,9 +19,7 @@
      */
     @ExpectPlatform
     public static <T extends BlockEntity> void register(
-            BlockEntityType<T> beType,
-            Function<BlockEntityRenderDispatcher, BlockEntityRenderer<T>> renderer
-    ) {
+        BlockEntityType<T> beType, Function<BlockEntityRenderDispatcher, BlockEntityRenderer<T>> renderer) {
```

Comment:
Idk how I feel about this

Reply:
hey ho guess what my opinion is: single line signatures :)